tests/suites/test_suite_aes.function

```diff
@@ -563,13 +563,21 @@ void aes_check_params( )
 /* BEGIN_CASE */
 void aes_misc_params( )
 {
+#if defined(MBEDTLS_CIPHER_MODE_CBC) || \
+    defined(MBEDTLS_CIPHER_MODE_XTS) || \
+    defined(MBEDTLS_CIPHER_MODE_CFB) || \
+    defined(MBEDTLS_CIPHER_MODE_OFB)
     mbedtls_aes_context aes_ctx;
+    const unsigned char in[16] = { 0 };
+    unsigned char out[16];
+#endif
 #if defined(MBEDTLS_CIPHER_MODE_XTS)
     mbedtls_aes_xts_context xts_ctx;
 #endif
-    const unsigned char in[16] = { 0 };
-    unsigned char out[16];
+#if defined(MBEDTLS_CIPHER_MODE_CFB) || \
+    defined(MBEDTLS_CIPHER_MODE_OFB)
     size_t size;
+#endif
```

Comment:
I think in this instance the code would be easier to read and maintain if we unconditionally included `((void) xxx)`, but that's of course only a comment and not a change request.